File: JAVA/Operators/Methods/src/PrimeNumber.java
public class PrimeNumber {
    static boolean primeOrNot(int x) {
        for (int i = 2; i < x/2; i++) {
            if (x % i == 0)
                return false;
        }
        return true;
    }
    public static void main(String args[]) {
        if (primeOrNot(19))
            System.out.println("Prime Number");
        else
            System.out.println("Not a Prime Number");
    }
}

File: Practice/prime_number.cpp
#include <iostream>
using namespace std;
int main()
{
    system("cls");
    int n;
    cout << "Enter a number : " << endl;
    cin >> n;
    int t;
    for (int i = 2; i < n / 2; i++)
    {
        if (n % i == 0)
        {
            cout << "Not a prime number" << endl;
            t = 0;
            break;
        }
    }
    if (t == 0)
        cout << "Number is Prime" << endl;

    return 0;
}